git-extra/notepad

```diff
@@ -15,7 +15,14 @@ die "Usage: $0 <file>"
 
 if test -f "$1"
 then
-	unix2dos.exe "$1"
+	commitencoding="$(git config --get i18n.commitencoding)"
+	if [ $? -ne 0 ]  ||
+		[ "$commitencoding" = "utf-8" ] ||
+		[ "$commitencoding" = "utf8" ]; then
+		unix2dos.exe -m "$1"
+	else
+		unix2dos.exe "$1"
+	fi
```

Comment:
Could you rewrite this as a

```sh
case "$(git config ...)" in
utf-8|utf8) ...
...
```

then run `updpkgsums` in `git-extra/` to update the checksums in `PKGBUILD`, and augment the commit message with a description of the problem that is solved?